textMining.py

```diff
@@ -0,0 +1,111 @@
+import wikipedia
+from vaderSentiment.vaderSentiment import SentimentIntensityAnalyzer
+import requests
+
+def wikiInfo():
+    """
+        Originally worked with wikipedia to look at online text
+
+        returns: tuple of text strings from wikipedia articles
+    """
+
+    # finds the pages
+    cat = wikipedia.page("Cat")
+    dog = wikipedia.page("Dog")
+
+    # saves the content of the pages as strings
+    catInfo = cat.content
```

Comment:
Official naming convention for variables and functions in python is lowercase letters separated by underscore if the name is more than one word. In this case, the variable names should be `cat_info` and `dog_info`.